pkg/server/proto/server.proto

```diff
@@ -4625,6 +4636,36 @@ message GetPipelineResponse {
   }
 }
 
+message RunPipelineRequest {
+  // The pipeline to execute.
+  Ref.Pipeline pipeline = 1;
+
+  // This is the job template to use for all the queued steps. This should
+  // set all the required fields of a job except for "operation".
+  Job job_template = 2;
+
+  // In the future, we'll put input variables and other options here.
+}
+
+message RunPipelineResponse {
+  // The root job of the pipeline.
+  string job_id = 1;
+
+  // All job IDs are the list of all jobs that were queued for this pipeline.
+  // They are in topologically sorted order. This means that a job's dependencies
+  // are guaranteed to be satisfied by previous jobs in the list, but all
+  // previous jobs may not be necessary. This is because two jobs may be
+  // totally independent of each other and can execute concurrently. To get
+  // a better view, a graph should be built using step dependencies, and
+  // the job IDs can be correlated using the job_map field.
+  repeated string all_job_ids = 2;
```

Comment:
Will `all_job_ids` include the root `job_id`? Would `AllJobIds[0] == JobId`? Mostly curious if that's something we could rely on, or if there's a good reason to keep the root job id in its own field.

Reply:
Yes, it will, but I thought for people who just want the root, separating it out would be cleaner in case we do every introduce the like... "get child jobs" API.